src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -194,7 +194,7 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
 
 .people-roles-summary-header {
   display: flex;
-  background-color: $gray1;
+  background-color: rgba(1, 51, 102, 0.15);
```

Comment:
I liked the old code better. Why did this change?

Reply:
Sorry, the theme colour was wrong. It is a header that is used in a few places but we hadn't established which theme colour it most closely resembles yet. I was updating this so it was the same colour used in the Step1 Review Header.

Reply:
I'll try to match to a theme colour and update this in step 1 and step 2 review headers 👍

Reply:
Or create a new colour that can be used elsewhere.

Reply:
Great idea!